CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Categories for each release: Added, Changed, Deprecated, Removed, Fixed, Securit
 
 * `dx extract_assay expression`
 * Return `jobLogsForwardingStatus` field in `dx describe --verbose job-xxxx`
+* Docker image can be cached on the platform
```

Comment:
1. Are users able to turn caching off? 
2. What if the docker image needs the applet runner to provide docker credential, but the applet builder does not have such access?

Reply:
1. Yes. Storage costs are negligible compared to amount of data produced by each run.
2. That's up to users to manage creds.